allauth/account/views.py

```diff
@@ -290,6 +291,19 @@ def get_context_data(self, **kwargs):
         )
         return ret
 
+    def get_initial(self):
+        intial = super(SignupView,self).get_initial()
```

Comment:
```suggestion
        intial = super(SignupView, self).get_initial()
```

To make black happy you need a space here.